### PR TITLE
net-irc/kvirc: update, unlastrite, take

### PR DESCRIPTION
**Pull request by DarthGandalf**
Closes: https://bugs.gentoo.org/905955

### CODE REVIEW

**Review by mattst88 — APPROVED** [review]
Great. Glad to have you take it over.

File: net-irc/kvirc/kvirc-5.2_pre20220629000000.ebuild [hunk 1]
```diff
@@ -0,0 +1,175 @@
+# Copyright 2009-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+CMAKE_MAKEFILE_GENERATOR="emake"
+PYTHON_COMPAT=( python3_{9..11} )
```
Comment (thesamesam) [inline net-irc/kvirc/kvirc-5.2_pre20220629000000.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{10..11} )
```

Comment (DarthGandalf) [reply net-irc/kvirc/kvirc-5.2_pre20220629000000.ebuild]: Done

**Comment by thesamesam** [comment]
Thank you!

**Pull request closed without merging**